CentralComputing/ADCManager.cpp

```diff
@@ -105,4 +110,31 @@ void ADCManager::initialize_sensor_error_configs() {
 }
 
 void ADCManager::check_for_sensor_error(const std::shared_ptr<ADCData> & check_data) {
+  //Just hardcoding and using difference for accelerometers for now
+  int32_t* adc_data = check_data->data;
+  if (abs(adc_data[0] - adc_data[1]) > error_accel_diff) {
```

Comment:
This is a bit tricky. Say we hit a bump and the accelerometers differ significantly for just 1 reading.  Realistically there is no cause for concern.

So we need to keep track of this in a different way. 
  * Could have a counter built in here that keeps track of the number of consistent instances where the accelerometers differ. After (maybe 10? 20?) of these errors, it triggers an error.
  * Could keep a rolling average of the last X values. If that average violates the `error_accel_diff`, then trigger an error